lib/elixir/test/elixir/kernel/docs_test.exs

```diff
@@ -300,6 +304,16 @@ defmodule Kernel.DocsTest do
     end
   end
 
+  if System.otp_release() >= "23" do
```

Comment:
Unfortunately this is not always true because the Erlang installation may have been compiled without chunks. I think for now it is best to emulate it. See the protocol_test for an example on how to write files to disk on tests and then also write a chunk file relative to that location.

Reply:
Good call, thank you, updated! Not sure what specific part of protocol_test you had in mind but I did my best, lmk!